pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BatchMessageTest.java

```diff
@@ -737,5 +737,69 @@ public void testOrderingOfKeyBasedBatchMessageContainer() throws PulsarClientExc
         producer.close();
     }
 
+    @Test(dataProvider = "containerBuilder", timeOut = 3000)
```

Comment:
```suggestion
    @Test(dataProvider = "containerBuilder")
```

Timeout is already enforced by default (and using a small timeout increases the chances of flaking when running in Jenkins).